MANIFEST.in

```
@@ -0,0 +1,6 @@
+include CHANGELOG.rst
+include LICENSE.txt
+include README.rst
+include requirements/base.in
+include requirements/constraints.txt
+recursive-include codejail-includes *.html *.png *.gif *.js *.css *.jpg *.jpeg *.svg *.py
```

`codejail-includes` is not a folder within this repo. Perhaps you need to include the sub-packages directly, like:

```suggestion
include eia.py
recursive-include verifiers *.html *.png *.gif *.js *.css *.jpg *.jpeg *.svg *.py
recursive-include loncapa *.html *.png *.gif *.js *.css *.jpg *.jpeg *.svg *.py
```